README.md

```diff
@@ -1,55 +1,56 @@
 # HoloToolkit-Unity
-This is effectively part of the existing HoloToolkit, but this is the repo that will contain all Unity specific components.
-The HoloToolkit is a collection of scripts and components intended to accelerate the development of holographic applications targeting Windows Holographic.
+This is effectively part of the existing HoloToolkit, but this is the repository that will contain all Unity specific components.
+The HoloToolkit is a collection of scripts and components intended to accelerate development of holographic applications targeting Windows Holographic.
 
----
-
-## Plugins
-Folder containing the native and managed binaries being compiled from the HoloToolkit repository.
-
----
+HoloToolkit contains the following feature areas:
 
-## Utilities
-Useful common concepts that you can leverage in your apps.
+1. [Input](https://github.com/NeerajW/HoloToolkit-Unity#input)
```

Comment:
These links go to your forked repo (https://github.com/NeerajW/HoloToolkit-Unity), but should link to the main Microsoft repository (https://github.com/Microsoft/HoloToolkit-Unity).